packages/ui/src/App.tsx

```diff
@@ -71,10 +73,18 @@ const mdTheme = createTheme({
 });
 
 function DashboardContent() {
-  const [open, setOpen] = React.useState(true);
+  const [open, setOpen] = useState(true);
+
+  useLayoutEffect(() => {
```

Comment:
`setLayoutEffect` renders the open state before anything is displayed. Otherwise there is a UI flash where the open state isn't correct before being set.